packages/validation/docs/validation-reports.md

```diff
@@ -0,0 +1,107 @@
+# Validation Reports
+
+## Restriction Levels
+Restrictions apply to multiple different levels, differentiated by what data is needed to check their rules.
+
+
+#### Dictionary
+Considers all records for all schemas of the dictionary.
+
+- ForeignKey
```

Comment:
is this initial capital on purpose? seems to break the snakeCase pattern, so just double checking